javascript/javascript-core/javascript-arrays/setting-the-length-of-an-array.md

```diff
@@ -5,6 +5,8 @@ category: hack
 tags:
   - introduction
   - obscura
+links:
+- '[Array holes](https://medium.com/better-programming/what-are-holes-in-arrays-3ac5fcbcd1c){website}'
```

Comment:
let's use this format for links since at one point in the future we'll want to make all the links like this
```suggestion

links:
  - >-
    [Array Holes](https://medium.com/better-programming/what-are-holes-in-arrays-3ac5fcbcd1c){website}
```